### ISSUE TITLE
KeyError: 'query_name'

### ISSUE THREAD

**Issue by MDSharma** [dib-lab/dammit#148]
Hi,

Trying to annotate the test fasta from the manual with the latest pipeline v1.1 gives the following error:

```
- [ ] remap_hmmer:longest_orfs.pep.x.pfam.tbl:
    * Python: function get_remap_hmmer_task.cmd
- [ ] hmmscan-gff3:cdna_nointrons_utrs.fa.x.pfam-A.gff3:
    * Cmd: `rm -f /nobackup/beegfs/home/ISAD/mds207/test/cdna_nointrons_utrs.fa.dammit/cdna_nointrons_utrs.fa.x.pfam-A.gff3`
    * Python: function get_hmmscan_gff3_task.cmd
########################################
TaskError - taskid:hmmscan-gff3:cdna_nointrons_utrs.fa.x.pfam-A.gff3
PythonAction Error
Traceback (most recent call last):
  File "/cm/shared/admin-apps/dammit/1.0.rc0/lib/python3.6/site-packages/pandas/core/indexes/base.py", line 2657, in get_loc
    return self._engine.get_loc(key)
  File "pandas/_libs/index.pyx", line 108, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 132, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 1601, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 1608, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'query_name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/cm/shared/admin-apps/dammit/1.0.rc0/lib/python3.6/site-packages/doit/action.py", line 383, in execute
    returned_value = self.py_callable(*self.args, **kwargs)
  File "/cm/shared/admin-apps/dammit/1.0.rc0/lib/python3.6/site-packages/dammit-1.1-py3.6.egg/dammit/tasks/gff.py", line 146, in cmd
    writer.write(group)
  File "/cm/shared/admin-apps/dammit/1.0.rc0/lib/python3.6/site-packages/dammit-1.1-py3.6.egg/dammit/fileio/gff3.py", line 260, in write
    data_df = self.convert(data_df)
  File "/cm/shared/admin-apps/dammit/1.0.rc0/lib/python3.6/site-packages/dammit-1.1-py3.6.egg/dammit/fileio/gff3.py", line 230, in convert
    return self.converter(data_df, **self.converter_kwds)
  File "/cm/shared/admin-apps/dammit/1.0.rc0/lib/python3.6/site-packages/dammit-1.1-py3.6.egg/dammit/fileio/gff3.py", line 143, in hmmscan_to_gff3
    gff3_df['seqid'] = hmmscan_df['query_name']
  File "/cm/shared/admin-apps/dammit/1.0.rc0/lib/python3.6/site-packages/pandas/core/frame.py", line 2927, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/cm/shared/admin-apps/dammit/1.0.rc0/lib/python3.6/site-packages/pandas/core/indexes/base.py", line 2659, in get_loc
    return self._engine.get_loc(self._maybe_cast_indexer(key))
  File "pandas/_libs/index.pyx", line 108, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 132, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 1601, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 1608, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'query_name'
```
Any thoughts @bluegenes ?

**Comment by davidecarlson** [comment]
Were you able to solve this?  I'm seeing a similar error.

**Comment by camillescott** [comment]
Sorry for the long wait! This is solved in v1.2, which is now on bioconda. You can get it with `conda install dammit=1.2`.

Cheers!
